### ISSUE TITLE
Support alternative content types in default handlers

### ISSUE THREAD

**Issue by jtc42** [labthings/python-thingserver#2]
Right now JSON output is hard coded. We should have provisions for returning other data types.

**Comment by jtc42** [comment]
Might be best to find a simple way for properties to define their content type, and then have a map of converter functions. Replace 

```
self.set_header("Content-Type", "application/json")
self.write(json.dumps(self.thing.get_property(property_name)))
```

With something that reads the content type and replaces the JSON dump with some other method that returns a string or bytes

**Comment by jtc42** [comment]
Closed by 3b3d29e38d0bce8f16c5b3c3a55450303ce51aa4
